### PR TITLE
fix: adds support for tel and other hyperlink protocols (DLC-228)

### PR DESCRIPTION
**Pull request by matheuspoleza**
### Brief description. What is this change?
We want to allow users to use tel, mailTo and other protocols as hyperlinks.

### CODE REVIEW

File: packages/react-chat/src/components/Text/schema.ts [hunk 1]
```diff
@@ -12,12 +12,14 @@ export const schema = {
   },
   protocols: {
     ...defaultSchema.protocols,
+    href: ['https', 'http', 'mailto', 'tel', 'sms', 'geo'], // Inclui 'geo'
```
Comment (ninabondar) [inline packages/react-chat/src/components/Text/schema.ts:15]: comment looks cropped

**Review by ninabondar — APPROVED** [review]

File: packages/react-chat/src/components/Text/schema.ts (hunk 1 above)
Comment (matheuspoleza) [reply packages/react-chat/src/components/Text/schema.ts]: portuguese debug comment lol

**Pull request merged**